library/bConnect/bConnect/Endpoints/EndpointOptions.cs

```diff
@@ -33,6 +38,11 @@ public enum EndpointOptions
         /// </summary>
         NeverUpdatePrimaryUser = 12,
 
+        /// <summary>
+        /// Waiting is for those without resolve
+        /// </summary>
+        AlwaysExecuteUserJobs = 0,
```

Comment:
Endpoint Options are a flag enumeration. With
- Bit 1 for OS Install
- Bit 2 for Auto Installation
- Bit 3-4 can be different Values, according to the PrimaryUserUpdateOptions
- Bit 5-6 can be different Values, according to the UserJobOptions

PrimaryUserUpdateOption can be on of the following values UpdatePrimaryUserOnNextLogon,DoNotUsePrimaryUser, AlwaysUpdatePrimaryUser, NeverUpdatePrimaryUser.

But i can use it in EndpointOptions only if all "suboptions" included in EndpointOptions.
Before the changes i couldnt use 'UpdatePrimaryUserOnNextLogon' because it was not included in EndpointOptions.